### DISCUSSION TITLE
how to connect with oracle or postgres or mysql database

### DISCUSSION THREAD

**Discussion by Aman78695** [vanna-ai/vanna#205] [Q&A]
i am using vanna jupyter notebook except that i am using this for connection...

db_user = "root"
db_password = "aman"
db_host = "localhost"
db_name = "jeevika"

cs=f"mysql+pymysql://{db_user}:{db_password}@{db_host}/{db_name}"

vn.connect_to_sqlite(cs)

...but i am getting this error....---------------------------------------------------------------------------
InvalidSchema                             Traceback (most recent call last)
[<ipython-input-7-3c7e8d59ab67>](https://localhost:8080/#) in <cell line: 1>()
----> 1 vn.connect_to_sqlite(cs)

5 frames
[/usr/local/lib/python3.10/dist-packages/requests/sessions.py](https://localhost:8080/#) in get_adapter(self, url)
    792 
    793         # Nothing matches :-/
--> 794         raise InvalidSchema(f"No connection adapters were found for {url!r}")
    795 
    796     def close(self):

InvalidSchema: No connection adapters were found for 'mysql+pymysql://root:aman@localhost/jeevika'

**Comment by zainhoda** [comment]
Please see the instructions here under "Other Database"
https://vanna.ai/docs/other-database-openai-vanna-vannadb/
